Src/SemanticComparison/ProxyGenerator.cs

```diff
@@ -173,13 +183,10 @@ private static void BuildMethodGetHashCode<TDestination>(TypeBuilder type)
             method.SetReturnType(typeof(int));
 
             int derivedGetHashCode = 135;
-            MethodInfo getHashCode = typeof(TDestination).GetMethod(

```

Comment:
Are you *sure* it's working now? It's been a while since I worked on the ProxyGenerator (*yes*, we use our own 😄) but I can tell you that this code can be ...sensitive 😆